### PR TITLE
Salamader CVar Changes

### PR DESCRIPTION
**Pull request by Just-a-Unity-Dev**
## About the PR

Changes some stuff for salamander

image

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

File: Resources/ConfigPresets/WizardsDen/salamander.toml [hunk 1]
```diff
@@ -11,6 +11,7 @@ rules_header = "ui-rules-header-rp"
 [whitelist]
 enabled = true
 reason = "whitelist-not-whitelisted-rp"
+min_players = 5
```
Comment (metalgearsloth) [inline Resources/ConfigPresets/WizardsDen/salamander.toml:14]: 5 is hilariously low, make it at least 20.

**Comment by Just-a-Unity-Dev** [comment]
holy shit i suck at naming

File: Resources/ConfigPresets/WizardsDen/salamander.toml (hunk 1 above)
Comment (Just-a-Unity-Dev) [reply Resources/ConfigPresets/WizardsDen/salamander.toml:14]: i'm happy to make it 10 but considering sala highpop is 20-25 i wouldn't prefer to do so

Comment (metalgearsloth) [reply Resources/ConfigPresets/WizardsDen/salamander.toml:14]: 20 is still considered lowpop in ss13.

**Comment by EmoGarbage404** [comment]
needs to be min 20, sloth is right

**Comment by mirrorcult** [comment]
yes make it 20

**Comment by DogZeroX** [comment]
Make it 20

**Comment by Nairodian** [comment]
I _personally_ disagree, we get around twenty regulars around right now, having at most twenty people who aren't whitelisted alongside those doesn't seem like a good idea to me. However, I suppose I'm alright with testing it.

**Comment by Just-a-Unity-Dev** [comment]
i concede

**Comment by Just-a-Unity-Dev** [comment]
however a lot of sala players don't, people from LRP *are* just going to join and grief cause they can. why not 10?

**Comment by mirrorcult** [comment]
20

**Comment by Emisse** [comment]
20

**Comment by EmoGarbage404** [comment]
rain has been strongarmed
the whole squad has commented
all we need is goodwheatley disapproval and we can merge.

**Comment by Just-a-Unity-Dev** [comment]
> rain has been strongarmed the whole squad has commented all we need is goodwheatley disapproval and we can merge.

wrong there's like 6 sala players that want it 10 or 15

**Comment by Nairodian** [comment]
A bigger number doesn't mean shit if the people who have devoted themselves to the server don't enjoy the state of it.

**Comment by Just-a-Unity-Dev** [comment]
also +20 pop happens for like 2 hours so the whitelist is essentially useless. might as well just disable it at that point

**Comment by EmoGarbage404** [comment]
> wrong there's like 6 sala players that want it 10 or 15

you think we care about players?

**Comment by its-helvi** [comment]
As a sala player/semi-regular, I personally really feel like this should be 10 and not 20.

**Comment by Just-a-Unity-Dev** [comment]
> > wrong there's like 6 sala players that want it 10 or 15
> 
> you think we care about players?

this PR is literally catered for the players what are you on lol

**Comment by metalgearsloth** [comment]
> also +20 pop happens for like 2 hours so the whitelist is essentially useless. might as well just disable it at that point

At AU peak the pop is consistently 0 https://ss14.qtqt.cf/hub/41

**Comment by mirrorcult** [comment]
> > rain has been strongarmed the whole squad has commented all we need is goodwheatley disapproval and we can merge.
> 
> wrong there's like 6 sala players that want it 10 or 15

theres like 6 sala players. i want people to actually play the server. the point of no whitelist below X players is to kickstart people joining

**Comment by Just-a-Unity-Dev** [comment]
> > > rain has been strongarmed the whole squad has commented all we need is goodwheatley disapproval and we can merge.
> > 
> > 
> > wrong there's like 6 sala players that want it 10 or 15
> 
> theres like 6 sala players. i want people to actually play the server. the point of no whitelist below X players is to kickstart people joining

pop has actually been peaking lately compared to a few months ago

**Comment by throge** [comment]
I play sal often. Its been getting a maxpop of about 20 on weekdays consistently for the past 2 weeks or so. A cap of 10 or 15 would be much more fitting.

**Comment by Scribbles0** [comment]
maybe this conversation should continue in #development?

**Comment by EmoGarbage404** [comment]
image
we have the data lol.
most of the time the server is empty at literally 0 pop.

**Comment by Just-a-Unity-Dev** [comment]
image
are you reading the monthly data?

**Comment by EmoGarbage404** [comment]
> image are you reading the monthly data?

this data is still majority under 5 pop

**Comment by throge** [comment]
> image we have the data lol. most of the time the server is empty at literally 0 pop.

Personally, I have no problem with sal being dead some hours of the day. I have to sleep sometime. I don't see salamander having potentially constant pop as a worthy trade off for a majority LRP playerbase.

**Comment by Just-a-Unity-Dev** [comment]
why doesnt github have a thistbh reaction wtf

**Comment by mirrorcult** [comment]
image
image
image

game admin chat msgs

**Comment by its-helvi** [comment]
So at low pop, around 0 players, no one's going to join anyways because there's no one else on, and at high pop, above the 20 player limit, people are going to get blocked by the whitelist. 
I feel like I missed something, how exactly does this help...?

**Comment by mirrorcult** [comment]
at low pop the potential base of players to draw from to kickstart the server is larger because it isn't restricted to whitelisted players

**Comment by throge** [comment]
The way I see it, we already have a shit ton of people whitelisted. Those who aren't whitelisted either haven't applied, and so aren't interested, or failed the whitelist, in which case they probably shouldn't be playing anyways. If we were to test it, wouldn't it be better to start small, then increase the maxcap rather than potentially ruin the experience and lose many of the regulars?

**Comment by Ugaluga** [comment]
Salamander is its own ecosystem. It's active for a bit, then it goes to sleep and the same goes for tomorrow. We have had the highest population in a good while right now. If you introduce LRP players into it, it will deter people from wanting to play it, as we are all quite engaged in the RP and it's very easy to fall out of the immersion. Thank you for your time.

**Comment by mirrorcult** [comment]
>  If we were to test it, wouldn't it be better to start small, then increase the maxcap

20 is small

**Comment by Just-a-Unity-Dev** [comment]
> > If we were to test it, wouldn't it be better to start small, then increase the maxcap
> 
> 20 is small

thats high with salamander numbers

**Comment by EmoGarbage404** [comment]
<20 approaches unplayable for most people.

**Comment by mirrorcult** [comment]
as i said before if you desperately miss the experience of playing with 4 other people in a game made for 30-100 players then you can start a private server or play on the other 2 servers we have that are usually empty

**Review by DrSmugleaf — APPROVED** [review]

**Comment by DrSmugleaf** [comment]
good talk everybody

**Comment by throge** [comment]
what

**Comment by mirrorcult** [comment]
if 20 sucks it'll go down to 15 or 10

**Comment by mirrorcult** [comment]
but i suspect it will be fine

**Comment by mirrorcult** [comment]
relaying more of my reasoning from discord

image
image
image
image
image

**Comment by DawBla** [comment]
> <20 approaches unplayable for most people.

Yeah, this

Managed to hop on Salamander recently with 22 people, sci didn't exist, we didn't have a captain or hop and the only member of sec was the brigmedic
On Spider/Miros we tried to make it work with 9 people and it was pretty miserable with most of the departments barely running if at all

You really do need like 30 people to make the station work

**Comment by deltanedas** [comment]
> why doesnt github have a thistbh reaction wtf

use :trollface: its the same

**Comment by GoodWheatley** [comment]
> all we need is goodwheatley disapproval and we can merge.

Don’t care, I have more fun playing on Lizard anyway

**Pull request merged**